spring-cloud-circuitbreaker-resilience4j/pom.xml

```diff
@@ -27,6 +27,11 @@
 			<groupId>io.github.resilience4j</groupId>
 			<artifactId>resilience4j-timelimiter</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>io.github.resilience4j</groupId>
+			<artifactId>resilience4j-all</artifactId>
+			<scope>provided</scope>
```

Comment:
shouldn't this be `optional`?

Reply:
Why not use `resilience4j-bulkhead`?

Reply:
yep, changed.

`resilience4j-all` provides a Decorator class which is very useful for creating citcuitBreater/TimeLimiter/Bulkheads pipeline. 
And `resilience4j-all` includes `resilience4j-bulkhead` module

Reply:
Do you know which module the Decorator is in?  `-all` brings in much more than we need.

Reply:
In `resilience4j-all`: [Decorators](https://github.com/resilience4j/resilience4j/blob/master/resilience4j-all/src/main/java/io/github/resilience4j/decorators/Decorators.java)

Reply:
@Ferioney how much extra code would it be to not use the decorators?  I talked this over with the team and the general consensus was that we should avoid requiring extra dependencies when they are not needed.

Reply:
I think we can do it without Decorators:
[Commit 870dc37 Resilience4jBulkheadProvider.java](https://github.com/spring-cloud/spring-cloud-circuitbreaker/pull/87/commits/870dc37bd1bacb0f6d52db5d00e4a79063b43b51#diff-6ddba7b121b4caa51fd8c5b41821a64ce0d7d52458b3ace890911c51d41d06feR108)